src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,159 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
```

Comment:
Don't use class Objects. Instead, make `Node<T>` an inner class and move it to the end of the `MyLinkedList` class.